apps/calendar/js/backend/calendar_service.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+importScripts('../ext/alameda.js');
+
+require.config({
+  baseUrl: '/js/service',
```

Comment:
are you sure this `baseUrl` is needed and is correct?

maybe it's better to set the `baseUrl: '/js/backend'` and `paths: { service: '/js/service' }` to avoid confusions